mautic/mautic
# [UX] Add a tooltip for "Is repeatable" when creating points actions

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This enhancement introduces a tooltip for the “Is Repeatable” option during the creation of points actions. It aims to demystify the function for users, especially those unfamiliar with the term. 

The tooltip explains that enabling this feature allows a lead to accumulate points each time they complete the action. Conversely, disabling it restricts the lead to a one-time score assignment. This addition ensures a more user-friendly interface, providing immediate guidance and improving overall user experience.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Points > Manage Actions
3. Create a new
4. Hover on Is repeatable

![image](https://github.com/mautic/mautic/assets/116097999/6391dcd1-d9c9-45ef-8b95-ec44d5ccda28)
